packages/rust/lsprotocol/Cargo.toml

```diff
@@ -19,3 +19,6 @@ serde_json = "1.0.93"
 serde_repr = "0.1.10"
 url = {version = "2.3.1", features = ["serde"]}
 rust_decimal = "1.29.1"
+
+[package.metadata.docs.rs]
+features = ["proposed"]
```

Comment:
@karthiknadig, given that the product also doesn't build without enabling the `proposed` feature, maybe we should set

```toml
[features]
default = ["proposed"]
```

Maybe this doc-specific change wouldn't be needed in that case?

Reply:
Does it even make sense to have a `proposed` feature at all if the crate won't build without it?